scipy/stats/_discrete_distns.py

```diff
@@ -251,10 +255,11 @@ class nbinom_gen(rv_discrete):
 
        f(k) = \binom{k+n-1}{n-1} p^n (1-p)^k
 
-    for :math:`k \ge 0`.
+    for :math:`k \ge 0`, :math:'0 \leq p \leq 1'
```

Comment:
Strict inequality on the low end:
```suggestion
    for :math:`k \ge 0`, :math:`0 < p \leq 1`
```

Reply:
If this requires a strict inequality, should the `_argcheck` also include  the strict inequality? for `nbinom_gen` it is as follows: 
```python    
 def _argcheck(self, n, p):
        return (n > 0) & (p >= 0) & (p <= 1)`

I guess if so that would be part of a different commit to address that.

Reply:
Good point, that should be `(p > 0)` (likewise for `geom`).  If you are interested, you can make those changes in this PR, or we can do that in a separate PR.

Reply:
Sure, I'll just add that to this one - that seems simple enough to include.